src/serviceTypes.ts

```diff
@@ -1,7 +1,10 @@
 import type { Connection } from 'mongoose';
 import type { BaseLogger } from 'pino';
 
+import type { InternetEndpoint } from './utilities/awala/InternetEndpoint.js';
+
 export interface ServiceOptions {
```

Comment:
I think this should go along with `example[.spec].ts`, which is the only thing that would use it.